FlowCrypt/Core/CoreTypes.swift

```diff
@@ -139,6 +139,7 @@ struct SendableMsg: Equatable {
     let replyToMimeMsg: String?
     let atts: [Attachment]
     let pubKeys: [String]?
+    let threadId: String?
```

Comment:
I don't think this one needs it. It would only be meaningful if the TS code received and used it on the other end, which afaik it doesn't.

Reply:
please try that - it would be confusing to have this be passed into composeEmail where it's not used

Reply:
`threadId` added as argument